src/domain/products/details/variants/variant-editor.tsx

```diff
@@ -79,6 +79,19 @@ const VariantEditor = ({
   }, [variant, store])
 
   const handleSave = (data) => {
+    if (!data.prices) {
+      const element = document.getElementById("add-price")
+      if (element) {
+        element.focus()
+      }
```

Comment:
```suggestion
      document.getElementById("add-price")?.focus()
```

Reply:
We also have a utility for focussing by name we could use. Up to you.
```suggestion
      focusByName("add-price")
```